src/worldcereal/openeo/postprocess.py

```diff
@@ -32,6 +33,139 @@ def output_labels(self) -> list:
     def dependencies(self) -> list:
         return []
 
+    @classmethod
+    def majority_vote(
```

Comment:
This is tested for multiclass arrays?

Reply:
yes, tested it for multiclass crop type...